src/machines/test_help/workflow_driver.rs

```diff
@@ -37,6 +39,9 @@ pub(in crate::machines) struct TestWfDriverCache {
     /// means keeping the workflow suspended in a future somewhere. I tried this and it was hard,
     /// but ultimately it's how real workflows will need to work.
     unblocked_timers: DashMap<String, bool>,
+
+    // Holds a mapping of activity id -> is completed
+    completed_activities: DashMap<String, bool>,
```

Comment:
Was hoping you wouldn't need to do this but looks like you got it working which is nice. I should be able to get rid of all this soon

Reply:
We can ditch it together with the test in the `activity_state_machines.rs`, since we've agreed to use the style similar to what is in the `lib.rs`

Reply:
As mentioned in slack, merge master and see TODO in new async workflow driver

Reply:
Done.